doc/man7/openssl-glossary.pod

```diff
@@ -30,11 +62,73 @@ It is defined in ITU-T document X.690:
 
 L<https://www.itu.int/rec/T-REC-X.690>
 
+=item Encoder
+
+An encoder is a type of algorithm used for encoding keys and parameters to some
+external format such as PEM or DER.
+
+L<OSSL_ENCODER_CTX_new_for_pkey(3)>
+
+=item Fetching
+
+Fetching is the process of looking through the available algorithm
+implementations, applying selection criteria (via a property query string), and
+finally choosing the implementation that will be used. Fetches can either be
+explicit where a function call is made to obtain an algorithm object
+representing an implementation such as L<EVP_MD_fetch(3)> or
+L<EVP_CIPHER_fetch(3)>, or implicit where an algorithm object with no associated
+implementation is used such as the return value from L<EVP_sha256(3)> or
+L<EVP_aes_128_cbc(3)>. With implicit fetching an implementation is fetched
+automatically using default selection criteria the first time the algorithm is
+used.
+
```

Comment:
Shouldn't we mention the engine/provider fetch order somewhere here?

Reply:
This seems to me like a small detail which doesn't really belong in the glossary.